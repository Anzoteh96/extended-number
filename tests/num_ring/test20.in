867280786526988036906374622442925641326644285405313404515141
204461235681546910178944512499345577730395941443984421398329
285875427438865852875867897316101007308665038716996750454850
