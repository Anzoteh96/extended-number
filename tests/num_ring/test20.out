Enter the number base b:
Enter two numbers n and m:
b is 867280786526988036906374622442925641326644285405313404515141
n mod b is 204461235681546910178944512499345577730395941443984421398329
m mod b is 285875427438865852875867897316101007308665038716996750454850
n + m is 490336663120412763054812409815446585039060980160981171853179
n - m is 785866594769669094209451237626170211748375188132301075458620
n * m is 104633608229741880219324076713388692275013371754311560230179
n ** m is 146167679108469243247856970141075310240663892422025378116957
